src/sentry/models/user.py

```diff
@@ -68,6 +68,15 @@ class User(BaseModel, AbstractBaseUser):
             'modifying their account (username, password, etc).'
         )
     )
+    is_sentry_app = models.BooleanField(
```

Comment:
We can't add this column without the field being NULLable and default to NULL. Feel free to grab me if you have specific questions about why or want to learn more.